src/ecosystem/Ecosystem.jl

```diff
@@ -207,6 +207,18 @@ function create_coral_struct(bounds::Tuple{Float64,Float64}=(0.9, 1.1))::Nothing
     return
 end
 
+"""
+    colony_areas(colony_diam_means)
+
+Generate colony areas for each size class.
+
+# Arguments
+- `colony_diam_means` : mean colony diameter (in meters)
+
+"""
+function colony_mean_area(colony_diam_means)
+    return pi .* ((colony_diam_means ./ 2.0).^2)
+end
```

Comment:
Function signature in the docstring doesn't match the implemented function? (L 211 compared to L 219)

Reply:
Oh I see, you copy pasted from below and forgot to update